packages/core/database/lib/query/query-builder.js

```diff
@@ -302,9 +302,15 @@ const createQueryBuilder = (uid, db) => {
         }
         case 'count': {
           const dbColumnName =
-            state.count === '*' ? '*' : this.aliasColumn(helpers.toColumnName(meta, state.count));
-
-          qb.count({ count: dbColumnName });
+            state.count === '*'
```

Comment:
this check seems a bit weird. we shouldn't transform * to id if the user wants to use *
I think you should change the default count to be 'id' (Line 74) and simplify the code here to always use state.count

Reply:
yeah that does make sense makes sense